internetarchive/cli/ia.py

```diff
@@ -64,7 +64,7 @@
 import sys
 
 from docopt import docopt, printable_usage
-from pkg_resources import DistributionNotFound, iter_entry_points
+from importlib.metadata import entry_points
```

Comment:
this needs a conditional import:
```suggestion
if sys.version_info < (3, 10):
  from importlib_metadata import entry_points
else:
  from importlib.metadata import entry_points
```

Reply:
Thanks for your suggestions. I've fixed the patch according to your suggestions.